website/payments/templates/payments/payment_list.html

```diff
@@ -63,12 +64,21 @@ <h1 class="text-center section-title">{% trans "payments"|capfirst %}</h1>
                             <td>€ {{ payment.amount }}</td>
                             <td>{{ payment.created_at }}</td>
                             <td class="text-left">{{ payment.notes }}</td>
+                            {% if payment.type == "tpay_payment" %}
```

Comment:
This is a whole lot of code just to do a small thing. For starters the html could be outside it. And you should look into the yesno template filter

Reply:
> Instead of blocking the form I would
> 
> * Disable the revoke button and show a message on hover using the tooltip functionality from bootstrap
> * Add the check inside the model's clean function

I thought of doing it in the model clean, but then I figured that as board member it might be nice to be actually able to register a 'valid until' date (as you do want to invalidate the mandate after all). Only